cmake/HighFiveWarnings.cmake

```
@@ -5,6 +5,8 @@ if(TARGET HighFiveWarnings)
     return()
 endif()
 
+message("HighFiveWanring")
```

Shouldnt the message be a bit more explanatory? Or dropped altogether?
```suggestion
message("HighFiveWarning")
```

---

Looks like debug output. This really is a sketch of WIP so that people can see what's meant by new CMake.